server/Database_queries/queries.js

```diff
@@ -25,15 +23,66 @@ const file_stats = async (user, bucketName, filename) => {
     return res;
   });
 };
+
 const file_uploaded = async (user, fileId) => {
   let query = `UPDATE FileTypeMap SET is_uploaded = ${1}  where file_unique_id = '${fileId}' AND user_name = '${user}';`;
   return await execSql(query).then(res => {
     return res;
   });
 };
 
-const delete_file = async (user, bucketName, fileId) => {
-  let query = `DELETE FROM FileTypeMap where file_unique_id = '${fileId}' AND bucket_name = '${bucketName}' AND user_name = '${user}';`;
+const delete_file = async (user, fileId) => {
+  let query = `DELETE FROM FileTypeMap where file_unique_id = '${fileId}' AND user_name = '${user}';`;
+  return await execSql(query).then(res => {
+    return res;
+  });
+};
+
+const add_user = async (user, userid) => {
+  let query = `INSERT INTO User_id_Map (user, user_id) VALUES ('${user}', '${userid}');`;
```

Comment:
your database table name is inconsistent with case

User_Id_Map or UserIdMap of userIdMap

Reply:
one doubt -> whats is the value of user here. and i assume user_id is user mail